src/main/java/io/odpf/kafkaclient/producer/ProducerFactory.java

```diff
@@ -1,6 +1,14 @@
 package io.odpf.kafkaclient.producer;
 
 import io.odpf.kafkaclient.ClientFactory;
+import io.odpf.kafkaclient.PropertiesNotSetException;
+import org.apache.kafka.clients.producer.KafkaProducer;
 
 public class ProducerFactory extends ClientFactory {
```

Comment:
Same as ConsumerFactory.

Reply:
Since you had asked in previous review to create separate class to replace Producer and Consumer constructor code.